notebooks/01-visualize-amounts-bar-plots.ipynb

```diff
@@ -0,0 +1,1384 @@
+{

```

Comment:
<p>suggest remove trailing punctuation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bokeh/dataviz-fundamentals/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Pls, which one is the trailing punctuation?</p>